alb.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_lb" "alb" {
   count                            = var.create_alb ? 1 : 0
-  name                             = "${var.prefix}-${var.cluster_name}-lb"
+  name                             = length("${var.prefix}-${var.cluster_name}-lb") > 33 ? format("%s-%s", substr(var.cluster_name, 0, 29), "lb") : "${var.prefix}-${var.cluster_name}-lb"
```

Comment:
you can just take always first 32 characters

Reply:
also no reason to `-lb` or `-lb-target-group` the only question is if it will re-create or change in place
if it changes in place lets also remove these suffixes

Reply:
I can not change the suffixes, the resources need to be recreated and we can't because it attaches to ec2, 
the max length is 32
I asked Jacky how to handle the case when we have more than 32 chars, and we agreed to use the cluster name because it is more important and makes the name clear the prefix